drop-in/src/main/java/com/adyen/checkout/dropin/DropIn.kt

```diff
@@ -256,6 +256,6 @@ object DropIn {
         } else {
             AdyenLogLevel.NONE
         }
-        AdyenLogger.setLogLevel(logLevel)
+        AdyenLogger.setInitialLogLevel(logLevel)
```

Comment:
Not in the scope of this PR but do you think we can do this for components as well?

Reply:
What if we use the app startup library for this? It could be a nice place to introduce this library 👀

Reply:
Will do that in a separate task